pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -600,6 +600,22 @@ func buildInboundClustersFromSidecar(cb *ClusterBuilder, proxy *model.Proxy,
 				if endpointAddress == "" {
 					endpointAddress = model.LocalhostIPv6AddressPrefix
 				}
+			} else if features.EnableDualStack {
```

Comment:
The next conditional clause would always use the first IP in `actualHosts`

Reply:
I am not sure why this has to be under dual stack

Reply:
Is this because one of the IPModes can be model.Dual? I'm _assuming_ that dual stack not enabled won't allow for this mode to exist? Not 100% sure how it works though..

Context: [wildCards](https://github.com/istio/istio/blob/6dd89f04b1b3ffe61f6ed2241fce12bcb2ed463e/pilot/pkg/networking/core/v1alpha3/listener_address.go#L45) and [localHosts](https://github.com/istio/istio/blob/6dd89f04b1b3ffe61f6ed2241fce12bcb2ed463e/pilot/pkg/networking/core/v1alpha3/listener_address.go#L51).